src/Transition/CreateNewIDFUsingRulesV9_3_0.f90

```diff
@@ -684,17 +684,18 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! Wait - there's more for the transition for AirTerminal:SingleDuct:Uncontrolled
               CASE('ROOMAIR:NODE:AIRFLOWNETWORK:HVACEQUIPMENT')
               IF(TotATSDUObjs > 0) THEN
+                 NoDiff = .true.
                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
-                 OutArgs(1) = InArgs(1)
-                 ! Loop through fields looking for  AirTerminal:SingleDuct:Uncontrolled node name
-                 DO nodeCount=2, CurArgs
-                   IF (SameString(TRIM(InArgs(nodeCount)), TRIM(ATSDUNodeNames(atCount)))) THEN
-                     OutArgs(nodeCount) = TRIM(InArgs(nodeCount)) // ' ATInlet'
-                   ELSE
-                     OutArgs(nodeCount) = InArgs(nodeCount)
+                 OutArgs(1:CurArgs) = InArgs(1:CurArgs)
+                 ! Loop through "ZoneHVAC or Air Terminal Equipment Object Type" fields looking for
+                 ! "AirTerminal:SingleDuct:Uncontrolled" and replace with "AirTerminal:SingleDuct:ConstantVolume:NoReheat"
+                 ! Four extensible fields
+                 DO CurField=2, CurArgs-1, 4
+                   IF (SameString(TRIM(InArgs(CurField)),'AirTerminal:SingleDuct:Uncontrolled')) THEN
+                     OutArgs(CurField) = 'AirTerminal:SingleDuct:ConstantVolume:NoReheat'
+                     NoDiff = .false.
                    END IF
-                 ENDDO
-                NoDiff = .false.
+                 END DO
```

Comment:
Then check every "ZoneHVAC or Air Terminal Equipment Object Type", if ATU:SngleDuct:Uncontrolled => replace and set NoDiff = false